server/api/user/index.js

```diff
@@ -1,7 +1,30 @@
 import express from 'express';
-
+import controller from './user.controller';
 const router = express.Router();
+// TODO: Verify user-id with token
+
+// GET => all user expenses for selected pool
+// Cannot GET /api/users/5/expense-pools-users/4/expenses
+router.get('/:id/expense-pools-users/:exp-pool-id/expenses',
```

Comment:
That makes sense. I originally broke it out because with the many-to-many relationship, I believe sequelize is going to break this into a separate table